doc/OnlineDocs/contributed_packages/pynumero/installation.rst

```diff
@@ -27,6 +30,18 @@ Method 2
 If you want PyNumero support for the HSL solvers and you have an IPOPT compilation
 for your machine, you can build PyNumero using the build script ::
 
-  cd pyomo/contrib/pynumero/
-  python build.py -DBUILD_ASL=ON -DBUILD_MA27=ON -DIPOPT_DIR=<path/to/ipopt/build/>
+  python -m pyomo.contrib.pynumero.build -DBUILD_ASL=ON -DBUILD_MA27=ON -DIPOPT_DIR=<path/to/ipopt/build/>
+
+Method 3
+--------
+
+You can build the PyNumero libraries from source using `cmake`.  This
+generally works best when building from a source distribution of Pyomo.
+Assuming that you are starting in the root of the Pyomo source
+distribution, you can follow the normal CMake build process ::
 
+  mkdir build
+  cd build
+  ccmake ../pyomo/contrib/pynumero/src
```

Comment:
Typo: cmake

Reply:
Nope.  I actually meant `ccmake` (that is the text-mode cmake build configuration editor).

Reply:
Ah ok, that's a new one for me.